flytepropeller/pkg/controller/executors/execution_context.go

```diff
@@ -4,7 +4,7 @@ import (
 	"github.com/flyteorg/flyte/flytepropeller/pkg/apis/flyteworkflow/v1alpha1"
 )
 
-//go:generate mockery-v2 --case=underscore --with-expecter --name ExecutionContext --output=mocks
+//go:generate mockery -all -case=underscore
```

Comment:
@hamersaw would we want to set this to --with-expecter -all ?

Reply:
I reverted it to what it was previously. If we set `--with-expecter` then we need to refactor everywhere one of these mocks is used to have `EXPECT()`. For an idea of the breadth, this PR just removes the `EXPECT()` from the `ExecutionContext` mocks. So there would be 7x more updates. We can refactor this in another PR if it is important.